### PR TITLE
added foreward

### PR DESCRIPTION
**Pull request by ttimbers**

### CODE REVIEW

File: foreward-text.Rmd [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+contribution to this fast-growing literature, but given the wide availability of materials, a
+reader should be inclined to ask, ”What is the unique contribution of this book?” In order
+to answer that question it is useful to step back for a moment and consider the development
+of the field of data science over the past few years.
+
+When thinking about data science, it is important to consider two questions: “What is
+data science?” and “How should one do data science?” The former question is under active
+discussion amongst a broad community of researchers and practitioners and there does
+not appear to be much consensus to date. However, there seems a general understanding
+that data science focuses on the more “active” elements—data wrangling, cleaning, and
```
Comment (trevorcampbell) [inline foreward-text.Rmd:22]: `elements—data wrangling` avoid unicode mdash, use `&mdash;` here (with no spaces between that and the words)

File: foreward-text.Rmd [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+analysis—of answering questions with data. These elements are often highly problem-
```
Comment (trevorcampbell) [inline foreward-text.Rmd:23]: unicode mdash again

Comment (trevorcampbell) [inline foreward-text.Rmd:23]: the hyphen here in "problem-specific" has broken over a newline

Please check for this kind of copy-paste error throughout, assuming you just copy-pasted from the pdf

File: foreward-text.Rmd [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+is and what the implications are for the activities in which members of the field engage.
+
+The first important concept addressed by this book is tidy data, which is a format for
+tabular data formally introduced to the statistical community in a 2014 paper by Hadley
+Wickham. The tidy data organization strategy has proven a powerful abstract concept for
+conducting data analysis, in large part because of the vast toolchain implemented in the
+Tidyverse collection of R packages. The second key concept is the development of workflows
+for reproducible and auditable data analyses. Modern data analyses have only grown in
+complexity due to the availability of data and the ease with which we can implement complex
+data analysis procedures. Furthermore, these data analyses are often part of decision-
```
Comment (trevorcampbell) [inline foreward-text.Rmd:37]: bad hyphen linebreak again. should be "decision-making processes"

File: foreward-text.Rmd [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+making processes that may have significant impacts on people and communities. Therefore,
+there is a critical need to build reproducible analyses that can be studied and repeated by
+others in a reliable manner. Statistical methods clearly represent an important element
+of data science for building prediction and classification models and for making inferences
+about unobserved populations. Finally, because a field can succeed only if it fosters an
+active and collaborative community, it has become clear that being fluent in the tools of
+collaboration is a core element of data science.
+
+This book takes these core concepts and focuses on how one can apply them to do data
```
Comment (trevorcampbell) [inline foreward-text.Rmd:46]: `do` should be `*do*` to match emphasis in the pdf

File: index.Rmd [hunk 5]
```diff
@@ -34,6 +34,9 @@ output:
 
 ---
 
+```{r preface, child="foreward-text.Rmd"}
```
Comment (trevorcampbell) [inline index.Rmd:37]: rename `foreward-text.Rmd` to `foreword-text.Rmd`

File: pdf/index.Rmd [hunk 6]
```diff
@@ -28,6 +28,9 @@ knitr::opts_chunk$set(fig.pos = "H",
 
 ```
 
+```{r preface, child="foreward-text.Rmd"}
```
Comment (trevorcampbell) [inline pdf/index.Rmd:31]: rename `foreward-text.Rmd` to `foreword-text.Rmd`

File: foreward-text.Rmd [hunk 7]
```diff
@@ -0,0 +1,63 @@
+# Foreward {-}
```
Comment (trevorcampbell) [inline foreward-text.Rmd:1]: filename should be `foreword-text.Rmd` (not `foreward-text.Rmd`)

**Review by trevorcampbell — CHANGES_REQUESTED** [review]

File: foreward-text.Rmd (hunk 7 above)
[the remark this answers is not in the document]
Comment (ttimbers) [reply foreward-text.Rmd:1]: Goodness...

Comment (ttimbers) [inline foreward-text.Rmd:1]:
```suggestion
# Foreword {-}
```

File: foreward-text.Rmd [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+Roger D. Peng
+
+*Johns Hopkins Bloomberg School of Public Health*
+
+January 4, 2022
```
Comment (ttimbers) [inline foreward-text.Rmd:7]:
```suggestion
*Roger D. Peng*

*Johns Hopkins Bloomberg School of Public Health*

*2022-01-04*
```

File: foreward-text.Rmd [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+The field of data science has expanded and grown significantly in recent years, 
+attracting excitement and interest from many different directions. The demand for introductory
+educational materials has grown concurrently with the growth of the field itself, leading to
+a proliferation of textbooks, courses, blog posts, and tutorials. This book is an important
+contribution to this fast-growing literature, but given the wide availability of materials, a
+reader should be inclined to ask, ”What is the unique contribution of this book?” In order
```
Comment (ttimbers) [inline foreward-text.Rmd:14]:
```suggestion
reader should be inclined to ask, "What is the unique contribution of this book?" In order
```

File: foreward-text.Rmd [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+to answer that question it is useful to step back for a moment and consider the development
+of the field of data science over the past few years.
+
+When thinking about data science, it is important to consider two questions: “What is
```
Comment (ttimbers) [inline foreward-text.Rmd:18]:
```suggestion
When thinking about data science, it is important to consider two questions: "What is
```

File: foreward-text.Rmd [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+data science?” and “How should one do data science?” The former question is under active
```
Comment (ttimbers) [inline foreward-text.Rmd:19]:
```suggestion
data science?" and "How should one do data science?" The former question is under active
```

File: foreward-text.Rmd (hunk 1 above)
Comment (ttimbers) [inline foreward-text.Rmd:22]:
```suggestion
that data science focuses on the more "active" elements—data wrangling, cleaning, and
```

File: foreward-text.Rmd (hunk 2 above)
Comment (ttimbers) [inline foreward-text.Rmd:23]:
```suggestion
analysis&mdash;of answering questions with data. These elements are often highly
```

File: foreward-text.Rmd [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+The first important concept addressed by this book is tidy data, which is a format for
+tabular data formally introduced to the statistical community in a 2014 paper by Hadley
+Wickham. The tidy data organization strategy has proven a powerful abstract concept for
+conducting data analysis, in large part because of the vast toolchain implemented in the
+Tidyverse collection of R packages. The second key concept is the development of workflows
+for reproducible and auditable data analyses. Modern data analyses have only grown in
+complexity due to the availability of data and the ease with which we can implement complex
+data analysis procedures. Furthermore, these data analyses are often part of decision-
+making processes that may have significant impacts on people and communities. Therefore,
```
Comment (ttimbers) [inline foreward-text.Rmd:38]:
```suggestion
data analysis procedures. Furthermore, these data analyses are often part of 
decision-making processes that may have significant impacts on people and communities. Therefore,
```

File: foreward-text.Rmd [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+to answer that question it is useful to step back for a moment and consider the development
+of the field of data science over the past few years.
+
+When thinking about data science, it is important to consider two questions: “What is
+data science?” and “How should one do data science?” The former question is under active
+discussion amongst a broad community of researchers and practitioners and there does
+not appear to be much consensus to date. However, there seems a general understanding
+that data science focuses on the more “active” elements—data wrangling, cleaning, and
+analysis—of answering questions with data. These elements are often highly problem-
+specific and may seem difficult to generalize across applications. Nevertheless, over time we
```
Comment (ttimbers) [inline foreward-text.Rmd:24]:
```suggestion
problem-specific and may seem difficult to generalize across applications. Nevertheless, over time we
```

File: foreward-text.Rmd (hunk 4 above)
Comment (ttimbers) [inline foreward-text.Rmd:46]:
```suggestion
This book takes these core concepts and focuses on how one can apply them to *do* data
```

File: index.Rmd (hunk 5 above)
Comment (ttimbers) [inline index.Rmd:37]:
```suggestion
```{r preface, child="foreword-text.Rmd"}
```

File: pdf/index.Rmd (hunk 6 above)
Comment (ttimbers) [inline pdf/index.Rmd:31]:
```suggestion
```{r preface, child="foreword-text.Rmd"}
```

**Review by ttimbers — COMMENTED** [review]
Small changes suggested. Will commit them next and finally rename the filename that was misnamed.

**Comment by ttimbers** [comment]
@trevorcampbell - I think I made all the changes you suggested!

**Pull request merged**